app/views/users/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="container">
+  <div class="user_info">
+    <div class="img">
+      <%= image_tag @user.photo, alt: "User Photo", class: "user-photo"%>
+    </div>
+    <div class="user">
+      <h2><%= @user.name %></h2>
+      <p><strong>Number of posts: </strong><%= @user.posts.count %></p>
+    </div>
+  </div>
+  <div class="user_bio">
+    <p>Bio</p>
+    <p><%= @user.bio %></p>
+  </div>
+  <% @user.recent_posts.each_with_index do |post, index| %>
+    <div class="post">
+      <h3>Post <%= index+1 %></h3>
+      <p><%= post.title %></p>
```

Comment:
- [ ] As stated in the requirements, users should be able to navigate between pages. This includes embedding links in the views. Kindly note that the post on the user show page should be a link that leads to the post.

Reply:
Fixed.